src/layouts/Unauthenticated.vue

```diff
@@ -11,4 +11,20 @@ export default {
 };
 </script>
 
-<style></style>
+<style>
+html,
+body,
+.wrapper,
+#app {
+  height: 100%;
```

Comment:
can you use `@apply w-full h-full` here?

Reply:
Definitely. As you can probably tell, not quite as confident yet in Grid as I am flex haha. Got a solid grasp of it now, but also just noticed that [tailwind has an entire section dedicated to grid helpers](https://tailwindcss.com/docs/grid-template-columns)... so probably will be redoing some of this to utilize those.

Reply:
Actually I think their grid is flex based. I think the way you're doing the grid is good. I'm just saying you can use the apply to use the tailwind height/width helpers

Reply:
Oh really? Had not looked into it much yet. In that case sounds good then.

Working on getting the basic phone API merged today, then will try and finish this up.